lib/edgehog_device_forwarder/supervisors/caches.ex

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule EdgehogDeviceForwarder.Supervisors.Caches do
```

Comment:
`Supervisors` is not a sensible context, Supervisors are an implementation details so this should be switched around instead: `EdgehogDeviceForwarder.Caching.Supervisor`

Reply:
I've gone for `EdgehogDeviceForwarder.Caches`